apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/ClaimType.java

```diff
@@ -17,6 +17,7 @@
 import gov.cms.bfd.model.rif.PartDEvent_;
 import gov.cms.bfd.model.rif.SNFClaim;
 import gov.cms.bfd.model.rif.SNFClaim_;
+import gov.cms.bfd.server.war.r4.providers.R4ExplanationOfBenefitResourceProvider;
```

Comment:
I understand your desire to add this; however, ultimately it is there based solely on a Javadoc link. So I think it best to refactor the javadoc without the link and remove this import.

Reply:
Sounds good. I'll make the necessary adjustments.